tests/ops_test.cc

```diff
@@ -592,6 +593,74 @@ TEST_P(OpDeviceTest, ReLU) {
   expect_storage_eq(output, expected);
 }
 
+TEST_P(OpDeviceTest, Log) {
+  Device device = GetParam();
+  float abs_diff = 0;
+#ifdef WITH_MKL
+  // the result of Log between vmsLn(WITH_MKL) and std::log has a bit different.
+  if(device == Device::CPU) {
+    abs_diff = 1e-6;
```

Comment:
I suggest to always set this absolute difference. I think this is also needed for GPU.

Reply:
Done